internal/compiler-bridge/src-2.10/main/scala/xsbt/DelegatingReporter.scala

```diff
@@ -5,18 +5,48 @@ package xsbt
 
 import xsbti.{ F0, Logger, Maybe }
 import java.io.File
+import java.util.Optional
```

Comment:
This is tracking the change introduced in https://github.com/sbt/util/pull/71/commits/f76e3aa2bb27dd82c53d15acb4907b01da31455a.
There are parts of sbt that is backed by Java API and by convention they are  put in `xsbti` package. We used to roll out own `Optional` called `xsbti.Maybe` but I am gradually switching over to `java.util.Optional` since I can now assume JDK8. `class PositionImpl` just implements `xsbti. Position`.